Source/WebKit/WebProcess/wpe/WebProcessMainWPE.cpp

```diff
@@ -51,6 +52,8 @@ class WebProcessMain final : public ChildProcessMainBase {
         // FIXME: This should be probably called in other processes as well.
         g_set_prgname("WPEWebProcess");
 
+        odh_error_report_init("WebKitBrowser");
```

Comment:
we should not initialize also network process? Please also send some test raport from Network process